css/style.css

```diff
@@ -1,19 +1,171 @@
-
- /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans+Condensed:wght@300&family=Rajdhani:wght@300&display=swap');
+/* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
 
 body {
-    font-family: 'Roboto', sans-serif;
-    -webkit-font-smoothing: antialiased;
+font-family: 'Roboto', sans-serif;
+-webkit-font-smoothing: antialiased;
 }
 
+
 /**
- * Add your custom styles below
- * 
- * Remember: 
- * - Be organised, use comments and separate your styles into meaningful chunks
- *    for example: General styles, Navigation styles, Hero styles, Footer etc.
- * 
- * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
- */
+* Add your custom styles below
+* 
+* Remember: 
+* - Be organised, use comments and separate your styles into meaningful chunks
+*    for example: General styles, Navigation styles, Hero styles, Footer etc.
+* 
+* - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
+*/
+
+
+/*Header,Logo and Menu Section*/
+.logo{
+width:50px;
+height: 50px;
+
+}
+
+.top_header{
+display: flex;
+width: 90%;
+margin: 0 auto;
+    
+}
 
+/*Top Menu*/
+.top_nav_menu{
+display: flex;
+justify-content: flex-end ;
+align-items: flex-end;
+padding-right: 80px;
+}
+
+.top_nav_menu li{
+padding: 15px;
+list-style: none;
+}
+
+.top_nav_menu li a{
+    text-decoration: none;
+    color: gray;
+}
+
+.top_nav_menu li a:hover{
+color: orange;
+transition: 0.4s;
+}
+/* nav Tag that holds menu items*/
+.menu_bar{
+flex: 2;
+}
+
+
+
+/* Background Image */
+.image_section{
+text-align: center;  
+position: relative; 
+padding-bottom: 90px;
+}
+
+/*container that hold Text and button link over the image*/
+.image_text_button{
+position: absolute;
+top: 30%;
+left: 30%;
+transform: translate(-30%, -30%);
+}
```

Comment:
It's best to avoid using `position: absolute` unless it's really necessary - because it can cause complications with layouts. 
In this case, you don't need to use it. If you set the parent element (`.image_section`) to `display: flex` and add flex rules to make the content centered, plus a bit of padding inside - you'll see your section should grow and have the text nice and centered.

Reply:
(this will work once you remove your `img` - see comment in your HTML file)